sharding-proxy/sharding-proxy-transport/sharding-proxy-transport-postgresql/src/main/java/org/apache/shardingsphere/shardingproxy/transport/postgresql/packet/generic/PostgreSQLErrorResponsePacket.java

```diff
@@ -22,20 +22,78 @@
 import org.apache.shardingsphere.shardingproxy.transport.postgresql.packet.command.PostgreSQLCommandPacketType;
 import org.apache.shardingsphere.shardingproxy.transport.postgresql.payload.PostgreSQLPacketPayload;
 
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * Error response packet for PostgreSQL.
+ * https://www.postgresql.org/docs/12/protocol-message-formats.html
  *
  * @author zhangyonglun
+ * @author avalon566
  */
 public final class PostgreSQLErrorResponsePacket implements PostgreSQLPacket {
-    
+
+    public static final char FIELD_TYPE_SEVERITY = 'S';
+
+    /**
```

Comment:
remove the following 3 lines

Reply:
I don't quite understand?

Reply:
I comment on line 39, I mean remove 3 lines from 39, i.e. remove line 39, 40, 41.

Reply:
No comments required?